### ISSUE TITLE
Danfoss LC-13 not setting temperature

### ISSUE THREAD

**Issue by darek** [AlCalzone/node-zwave-js#5241]
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within Z-Wave JS UI (formerly ZwaveJS2MQTT)?

NO, my problem is NOT within Z-Wave JS UI

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

What causes the bug?
Last weekend i have moved from Smartthings to HA, and I'm currently moving my Zwave devices (got Aeotec 7 stick with latest firmware). Inclusion is fine, i see all parameters in dashboard but I'm unable to set desired temperature. 
In Dashboard i see device is always in "sleep" mode but from time to time there is change in `last active` column and some events are sent (get temp and battery). In addition after some time device shows error code E5.

What do you observe?
It seems that on wakeup new values are not sent but driver gets old value and resend it to device.
In addition only for those devices i see additional warning:
```
2022-12-05T17:54:52.167Z DRIVER Received a command that contains multiple CommandClasses. This is not supporte
d yet! Discarding the message...
2022-12-05T17:54:52.167Z CNTRLR « [Node 005] TODO: no handler for application command
```

I was also digging into sources and i found out that device configs in file lc-13.json i found that lines:
```
"compat": {
		// If we don't query specific things on wake up, this device will
		// think it is not connected and show error E5
		"queryOnWakeup": [
			// ["CommandClass", "API method", ...arguments]
			["Battery", "get"],
			["Thermostat Setpoint", "get", 1]
		]
	}
```

It seems this fix does not work because i got E5 on all of my devices (4 of them), and I'm not sure how it works but maybe `["Thermostat Setpoint", "get", 1]` is polling value from thermostat and replacing my new setting with value from this query?

What did you expect to happen?
When setting specific temperature on HA Dashboard or directly in ZwaveJS-UI setting should be sent to TRV.

  1. Go to `Dashboard` where heating tile is set
  2. Click on `Tile` and set desired temperature
  3. Wait until TRV wake up 
  4. See refresh page and see old value
  5. Check debug log in `ZwaveJS-UI` and see entry like `Z-WAVE: Node 9: value updated: 67-0-setpoint-1 21 => 21`

### Device information

Manufacturer: Danfoss
Model name: LC-13
Node ID in your network: 9

### How are you using `node-zwave-js`?

- [ ] `zwave-js-ui` (formerly `zwavejs2mqtt`) Docker image (latest)
- [ ] `zwave-js-ui` (formerly `zwavejs2mqtt`) Docker image (dev)
- [ ] `zwave-js-ui` (formerly `zwavejs2mqtt`) Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

version:
`node-zwave-js` version: 1.3.0
`zwave-js-ui` version: 1.3.0

### Did you change anything?

yes (please describe)

### Did this work before?

Don't know, this is a new device

### Attach Driver Logfile

[zwavejs_2022-12-05.log](https://github.com/zwave-js/node-zwave-js/files/10157402/zwavejs_2022-12-05.log)

**Comment by darek** [comment]
I found that there is already issue with same stick i've got https://github.com/zwave-js/node-zwave-js/issues/4352 so i read comments and try to set power levels. I used 4 different values 0, 3, 6 and 10 (for both normal and output power).
I'm attaching log from todays test. What i found that my zwave devices created mesh and they are awaking from time to time to perform actions but for example i set wakeup time to 5 minutes for one device and its waking everery time if there is no E5 error on device but if it is its still awake but it take far more time .

Another thing if I set new value from HA and push wakeup button (little O on valve) it is able to get this temp and set it correctly.

I had also run Heal Network multiple times, but all valves finish healing only when i push O button.
[zwavejs_current.log](https://github.com/zwave-js/node-zwave-js/files/10178269/zwavejs_current.log)
